packages/api/models/user.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class user extends Model {}
```

Comment:
I think we'd still want the class name (here, on line 6, and line 17) to be capitalized, as that's pretty standard. It's the db name (in the migrations file) that we want to be lower case.